packages/cashier/src/stores/payment-agent-store.ts

```diff
@@ -201,20 +211,20 @@ export default class PaymentAgentStore {
     }
 
     @action.bound
-    addPaymentAgent(payment_agent) {
+    addPaymentAgent(payment_agent: TExtendedPaymentAgentList[0]): void {
         this.agents.push({
             text: payment_agent.name,
             value: payment_agent.paymentagent_loginid,
             max_withdrawal: payment_agent.max_withdrawal,
             min_withdrawal: payment_agent.min_withdrawal,
             email: payment_agent.email,
-            phone: payment_agent.telephone,
-            url: payment_agent.url,
+            phone: payment_agent.phone_numbers,
```

Comment:
according current BE:
payment_agent.telephone changed to `payment_agent.phone_numbers`
payment_agent.url changed to `payment_agent.urls`